property.go

```diff
@@ -20,6 +20,33 @@ type PropertyDefinitionConnection struct {
 	TotalCount int `graphql:"-"`
 }
 
+type PropertyInput struct {
+	Owner         IdentifierInput `json:"owner"`
+	Definition    IdentifierInput `json:"definition"`
+	Value         JSONString      `json:"value"`
+	RunValidation *bool           `json:"runValidation,omitempty"`
```

Comment:
is this actually exposed in the public API??

Reply:
Yes AFAICT
<img width="386" alt="image" src="https://github.com/OpsLevel/opslevel-go/assets/139589712/4b88290b-f5db-4702-be09-b698db96c68e">

Reply:
Talked to Foundations and they said that `runValidation` is expected to go public with the rest.